modules/admin/server/routes/admin.server.routes.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+var adminPolicy = require('../policies/admin.server.policy'),
+	admin = require('../controllers/admin.server.controller');
+
+module.exports = function(app) {
+	// Users collection routes
+	app.route('/api/users').all(adminPolicy.isAllowed)
```

Comment:
I understand what @mleanos is trying to say - we basically put all the admin functionalities in a central place, and when the admin capabilities will grow with time, as well as in our own personal projects it might create clutter to keep up to date or merge changes.